src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -27,21 +27,23 @@ export type ExportAnnotationsProps = {
 };
 
 type ExportFormat = {
-  value: 'json' | 'csv' | 'text' | 'html';
+  value: 'json' | 'csv' | 'txt' | 'html';
```

Comment:
This change allows using the value as file extension.

Reply:
That's OK, but probably worth noting in a comment that this is both an identifier _and_ file extension.